### PR TITLE
Ellipsis component

### PR DESCRIPTION
**Pull request by pr1moz**
Moved from the main repo.

### CODE REVIEW

**Comment by zenbot-0** [comment]
🔬 [Click here](https://reciprocity.github.io/zenlib/branches/Add-ellipsis) to test your changes on Storybook

File: packages/ellipsis/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# ColorInput
```
Comment (leferreyra) [inline packages/ellipsis/README.md]:
```suggestion
# Ellipsis
```

File: packages/ellipsis/Ellipsis.vue [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    <div class="content">
+      <slot />
+    </div>
+    <a href="#" class="ellipsis-toggle" @click.prevent="toggle" />
+  </div>
+</template>
+
+<script>
+export default {
+  name: "ZenEllipsis",
```
Comment (leferreyra) [inline packages/ellipsis/Ellipsis.vue]:
```suggestion
  name: "Ellipsis",
```

**Comment by pr1moz** [comment]
@leferreyra yeah, that line issue is due to this example case not being used inside a table. And we currently have a discrepancy in line-height between a table and outside. This will get fixed in the future, but for now, this is set for usage in a table (part of Listing, next PR).

**Comment by leferreyra** [comment]
@pr1moz right. IMHO if this is gonna be in ZenLib we have to make sure that it works standalone too, otherwise, it is just table specific.

This is the reason I don't like any global styles like bootstrap, it's super hard to know what the difference is between storybook and ZenGRC for example.

Isn't there a way we could make it work on all cases? I'm thinking instead of a fixed 2 lines we could pass the collapsed height as a prop? Idk

**Comment by pr1moz** [comment]
I really wouldn't prioritize other's usage of this, I think our time can be better spent elsewhere. As far as the table/other usage is concerned, I agree, but we first need to do some work on the ZenGRC project for that to be doable. And that is outside of this PR imo

**Comment by pr1moz** [comment]
I don't have time to do this now

**Comment by sagaban** [comment]
The idea behind the ZenLib poject was to have the component as much decoupled from ZebGRC as posible. Let's talk about this in the next FGM

**Comment by pr1moz** [comment]
@sagaban Then it doesn't make sense to move the ZenTable or Listing here anyway, because the presenters and data structure will be specific to ZenGRC.

**Comment by sagaban** [comment]
Oh, ok. In that case, doesn't make sense to have it here.

**Pull request closed without merging**